fix: log router_key in web/webpush web/simplepush

jrconlin opened the pull request:
closes: #745

bbangert commented on autopush/router/webpush.py:
```diff
@@ -132,6 +132,11 @@ def _save_notification(self, uaid, notification, month_table):
                                   headers={"TTL": str(notification.ttl),
                                            "Location": location},
                                   logged_status=204)
+        self.log.info("Saving message data",
```
We already log whether it was stored or online delivery in web/webpush.py, shouldn't the size and such be added there?

bbangert commented on autopush/web/webpush.py:
```diff
@@ -382,18 +385,25 @@ def _router_completed(self, response, uaid_data, warning=""):
             # No changes are requested by the bridge system, proceed as normal
             if response.status_code == 200 or response.logged_status == 200:
                 self.log.info(format="Successful delivery",
                               client_info=self._client_info)
```
Need all the data here as well.

jrconlin replied:
Cool, wasn't sure if you did.

pjenvey commented on autopush/web/webpush.py:
```diff
@@ -382,18 +385,25 @@ def _router_completed(self, response, uaid_data, warning=""):
-                              client_info=self._client_info)
+                self.log.info(
+                    format="Router miss, message stored.",
+                    message_size=len(notification.data) if notification else 0,
```
This was _client_info['message_size'] in endpoint. let's just dump the notification metadata in there up front so we don't have to the notification around

pjenvey commented on autopush/web/webpush.py:
```diff
@@ -382,18 +385,25 @@ def _router_completed(self, response, uaid_data, warning=""):
+                    format="Router miss, message stored.",
+                    message_size=len(notification.data) if notification else 0,
+                    ttl=notification.ttl if notification else 0,
+                    uaid=uaid_data['uaid'],
```
uaid should already be in client_info

bbangert commented:
Can we add the timing to the two log statements? It's not really feasible to get the data out of DataDog, so it'd be handy to include it with the message here (which would also let us easily differentiate time to store vs. online delivery).

jrconlin commented:
Added custom log observer (using a callback to do the actual testing).

Added route time to logging https://github.com/mozilla-services/autopush/pull/747/files#diff-ccbbd52ff880cf4ddb71b18d126d189cR369

bbangert commented on autopush/web/webpush.py:
```diff
@@ -382,18 +388,20 @@ def _router_completed(self, response, uaid_data, warning=""):
+                self.log.info(
+                    format="Router miss, message stored.",
+                    client_info=self._client_info)
             time_diff = time.time() - self.start_time
```
Why not just move this up and log this time?
